docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,343 @@
+# Prometheus Time Series Database Migration
+
+If you already have a running Kubernetes cluster and you are interested in installing the Bitnami Kubernetes Production Runtime (BKPR) on it, you may want to migrate the persisted data of your services with zero downtime and minimal data loss. In this case, we will talk about the Prometheus Time Series Database (TSDB), which contains the metrics that Prometheus scrapped from your services.
+
+This document describes the steps to migrate the TSDB from an existing Prometheus deployment to the one installed by BKPR.
+
+Furthermore, this process will not generate any downtime for your current Prometheus deployment, making it safe to be followed.
+
+## Pre-requisites
+
+The following prerequisites should be met before starting with the migration process:
+
+* You have a healthy Kubernetes cluster running.
+* You have a Prometheus deployment that *matches the BKPR Prometheus version*, which can be obtained from [the BKPR release compatibility table](https://github.com/bitnami/kube-prod-runtime#release-compatibility).
+* You have already read the [Kubeprod workflow documentation](https://github.com/bitnami/kube-prod-runtime/blob/a304cbc5b12042f2a17f96f23cd62a9759e81b4b/docs/workflow.md)
+* [Kubecfg binary](https://github.com/ksonnet/kubecfg/releases)
+
+
+This article will not cover the upgrade process of Prometheus itself. If you need to update your Prometheus deployment, check the [Prometheus docs](https://prometheus.io/docs/prometheus/latest/migration/)
```

Comment:
will not cover the Prometheus upgrade process.

Reply:
done